internal/core/items.rs

```diff
@@ -754,6 +754,25 @@ impl Item for FocusScope {
     }
 }
 
+impl FocusScope {
+    pub fn focus(self: Pin<&Self>, window_adapter: &Rc<dyn WindowAdapter>, self_rc: &ItemRc) {
+        WindowInner::from_pub(window_adapter.window()).set_focus_item(self_rc);
+    }
+}
+
+#[cfg(feature = "ffi")]
+#[no_mangle]
+pub unsafe extern "C" fn slint_focusscope_focus(
+    focus_scope: *const FocusScope,
```

Comment:
I think you can just use `Pin<&FocusScope>` here as the type since it is repr(transparent)

Reply:
Ohh, that would be nicer indeed.